app/models/reconciliation_deltas.rb

```diff
@@ -90,12 +90,7 @@ def changed_amount_css_class
 
     def changed_amount_icon
       return unless changed_amount?
-
-      if changed_amount > 0
-        %(<i class="glyphicon glyphicon-triangle-top"></i>).html_safe
-      else
-        %(<i class="glyphicon glyphicon-triangle-bottom"></i>).html_safe
-      end
+      changed_amount > 0 ? "top" : "bottom"
```

Comment:
Let's take this out and move this to a view helper method. Also, I wanted to point out that if we return nil in your current code we end up with a glyph icon that is non existent and that seems like a code smell.

Something like this perhaps:

```
module ReconciliationsHelper
  def changed_amount_icon(delta)
    return unless delta.changed_amount?

    if delta.changed_amount > 0
      tag(:i, class: "glyphicon glyphicon-triangle-top")
    else
      tag(:i, class: "glyphicon glyphicon-triangle-bottom")
    end
  end

  ...
end 
```